src/futils.c

```diff
@@ -21,32 +21,41 @@ int git_futils_mkpath2file(const char *file_path, const mode_t mode)
 		GIT_MKDIR_PATH | GIT_MKDIR_SKIP_LAST | GIT_MKDIR_VERIFY_DIR);
 }
 
+static void git_buf_put_rand(git_buf *buf, size_t chars)
+{
+	size_t i;
+	const char* alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+	for (i = 0; i < chars; ++i) {
```

Comment:
MInor formatting nit: we'd prefer no braces around a single line.

Reply:
Fixed(ish: by adding another line to the block)